cookbook/core/control_flow/chain_tasks.py

```diff
@@ -2,33 +2,30 @@
 Chain Flyte Tasks
 -----------------
 
-Data passing between tasks need not always happen through parameters.
+Data passing between tasks need not necessarily happen through parameters.
 Flyte provides a mechanism to chain tasks using the ``>>`` operator and the ``create_node`` function.
 You may want to call this function to specify dependencies between tasks that don't consume or produce outputs.
 
-In this example, let's enforce an order for ``read()`` to happen after ``write()``.
+Let's use this example to impose ``read()`` order after ``write()``.
```

Comment:
Can you revert this? I think the previous version looks good.

Reply:
Done.